qt/python/mantidqt/utils/writetosignal.py

```diff
@@ -36,7 +38,12 @@ def isatty(self):
         return False
 
     def write(self, txt):
-        # write to the console
-        self.__original_out.write(txt)
-        # emit the signal which will write to logging
+        if self.__original_out:
+            try:
+                self.__original_out.write(txt)
+            except IOError:
```

Comment:
Should we capture the exception object and append its message to the message given for easier debugging?